src/backend/src/prisma/migrations/20240325194935_add_ready_to_order_status/migration.sql

```diff
@@ -0,0 +1,18 @@
+/*
+  Warnings:
+
+  - The values [UNORDERED] on the enum `Material_Status` will be removed. If these variants are still used in the database, this will fail.
```

Comment:
You should be good to remove this whole warning comment block

Reply:
It's not removed